ftw/sliderblock/browser/resources/styles.scss

```diff
@@ -1 +1,11 @@
 @include portal-type-font-awesome-icon(ftw-sliderblock-sliderblock, forward);
+
+.ftw-sliderblock-sliderblock .inneredit {
+  font-size: $font-size-large;
+  line-height: $font-size-large;
+  position: absolute;
+  right: 0;
+
```

Comment:
The icon is often barely visible, depending on the image:
<img width="1201" alt="bildschirmfoto 2016-10-20 um 12 15 57" src="https://cloud.githubusercontent.com/assets/7469/19555952/2cb41e5e-96bf-11e6-87e0-032c203ad2ba.png">

I propose to use a ligther text color and apply a dark outer glow effect for having more contrast, making the icon better visible on most images, without having to apply a background color or moving the icon out of the image.
```css
  &:before {
    color: #EEE;
    text-shadow: 1px 0px 1px rgba(136, 136, 136, 0.7), 0px 1px 1px rgba(136, 136, 136, 0.7), -1px 0px 1px rgba(136, 136, 136, 0.7), 0px -1px 1px rgba(136, 136, 136, 0.7);
  }
  &:hover:before {
    color: #FFF;
    text-shadow: 1px 0px 1px rgba(136, 136, 136, 1), 0px 1px 1px rgba(136, 136, 136, 1), -1px 0px 1px rgba(136, 136, 136, 1), 0px -1px 1px rgba(136, 136, 136, 1);
  }
```
<img width="1207" alt="bildschirmfoto 2016-10-20 um 12 16 28" src="https://cloud.githubusercontent.com/assets/7469/19555971/44aea97a-96bf-11e6-9bd2-e0bc856acda1.png">